lib/DDG/Spice/Pokemon/Data.pm

```diff
@@ -21,9 +21,11 @@ spice alt_to => {
 	}
 };
 
+my $accepted_names = join "|", share('pokemon-names.txt')->slurp(chomp => 1);
+
 # Handle statement
 handle remainder => sub {
-    return lc $_ unless !$_ or $_ =~ /\s/;
+    return lc $_ unless !$_ or $_ =~ /\s/ or !($_ =~ /\b($accepted_names)\b/i);
```

Comment:
@moollaza Again, I am new to Perl (still learning). Thank you so much for introducing me to new & efficient ways to do things. :smile: